Users/coafile.rst

```diff
@@ -39,6 +39,11 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you usually don't want to lint,
+like minified files (e.g. ``*.min.js``) and backup files (e.g. ``*.orig``)::
+
+    ignore = *.min.js, *.orig
```

Comment:
hm not sure, maybe it should be `**.min.js, **.orig` (so irrespective of directory level, those endings are excluded).
Could you try that in coala? Try it on some empty file in some nested directories to see which files are grabbed by coala and which not :+1:

Reply:
I tried `**.min.js` i thought it worked the way u expected. guide me what to do further, should i change that?

Reply:
> i thought it worked the way u expected.

did it not?

Reply:
it worked, as expected.

Reply:
taht is why i made a pr changing `**.min.js`